service/config.go

```diff
@@ -13,6 +13,10 @@ type Config struct {
 	Listener net.Listener
 	// ProcessArgs are the arguments to launch a new process.
 	ProcessArgs []string
+	// Wd is working directory of the new process. This field is used only when
+	// ProcessArgs is specified.
```

Comment:
@aarzilli, if I am not wrong, you should be able to pass DEBUG_ONLY_THIS_PROCESS in syscall.SysProcAttr.CreationFlags. Would you like me to try and change code once this PR is submitted?